openscap_daemon/oscap_helpers.py

```diff
@@ -443,3 +443,24 @@ def get_status_from_exit_code(exit_code):
     "generate_report_for_result",
     "get_status_from_exit_code"
 ]
+
+
+def generate_fix_from_result(config, results_path):
+    if not os.path.exists(results_path):
+        raise RuntimeError("Can't generate fix for scan result. Expected "
+                           "results XML at '%s' but the file doesn't exist."
+                           % results_path)
+    try:
+        tree = ElementTree.parse(results_path)
+        root = tree.getroot()
+        ns = {"xccdf": "http://checklists.nist.gov/xccdf/1.2"}
+        test_result = root.find(".//xccdf:TestResult", ns)
+        result_id = test_result.attrib["id"]
+    except:
```

Comment:
Hate all catch statements, catch the exception you are looking for by type. This will catch AttributeErrors and other exceptions you definitely don't want to catch.